web/src/auth/LoginPage.js

```diff
@@ -302,13 +302,9 @@ class LoginPage extends React.Component {
           title="Sign Up Error"
           subTitle={"The application does not allow to sign up new account"}
           extra={[
-            <Link key="login" onClick={() => {
-              Setting.goToLogin(this, application);
-            }}>
-              <Button type="primary" key="signin">
-                Sign In
-              </Button>
-            </Link>,
+            <Button type="primary" key="signin">
```

Comment:
Link inside Button? Does the UI work right?

Reply:
UI works right.